src/jdk.compiler/share/classes/com/sun/tools/javac/main/JavaCompiler.java

```diff
@@ -232,6 +236,10 @@ else if (option.equals("class"))
      */
     public Log log;
 
+    /** Whether or not the options lint category was initially disabled
```

Comment:
You've copied my pattern of misuse.  "Whether or not" is not recommended use, whether or not you like it.
The consensus is to use just `whether`;  `whether or not` means `regardless`

FWIW, I still like the `whether or not` form!